cmd/flow-aggregator/config.go

```diff
@@ -51,4 +51,12 @@ type FlowAggregatorConfig struct {
 	// Provide format for records sent to the configured flow collector. Supported formats are IPFIX and JSON.
 	// Defaults to "IPFIX"
 	RecordFormat string `yaml:"recordFormat,omitempty"`
+	// RecordContents contains flow records related configuration options.
+	RecordContents RecordContentsConfig `yaml:"recordContents,omitempty"`
```

Comment:
Can we use simple type definition like featureGates in Antrea Agent? We can avoid a separate struct.
https://github.com/antrea-io/antrea/blob/main/cmd/antrea-agent/config.go#L23

Reply:
Thanks Srikar. Sounds good as we may only need boolean value in the RecordContents. Updated.